### PR TITLE
Fix error response retrieving certain cases

### PR DESCRIPTION
**Pull request by seallard**
## Description
Closes https://github.com/Clinical-Genomics/trailblazer/issues/342. Apparently, the `started_at` field for a job can be empty.

### Fixed
- Bug where searching for specific cases causes an error response

### How to test
- [x] deploy to stage
- [x] search for vitalsquid (not present in stage... Well this fix should work)

### This [version](https://semver.org/) is a
- [ ] **PATCH**

### CODE REVIEW

File: trailblazer/server/api.py [hunk 1]
```diff
@@ -108,9 +107,8 @@ def me():
 @blueprint.route("/aggregate/jobs")
 def aggregate_jobs():
     """Return stats about failed jobs."""
-    time_window: datetime = get_date_number_of_days_ago(
-        number_of_days_ago=int(request.args.get("days_back", ONE_MONTH_IN_DAYS))
-    )
+    number_of_days_ago: int = int(request.args.get("days_back", ONE_MONTH_IN_DAYS))
```
Comment (islean) [inline trailblazer/server/api.py:110]: Does this count as an instantiation of an `int` in that we can remove the type hint 🤔 Either way, you can disregard this comment

File: trailblazer/server/api.py [hunk 2]
```diff
@@ -186,10 +184,8 @@ def delete(analysis_id):
 def post_get_latest_analysis():
     """Return latest analysis entry for specified case id."""
     post_request: Response.json = request.json
```
Comment (islean) [inline trailblazer/server/api.py:186]: How come the `post_request` is a `Response.json` object?

**Review by islean — APPROVED** [review]
Love all the walruses 🦭 (closest I could find)

File: trailblazer/server/api.py (hunk 2 above)
Comment (seallard) [reply trailblazer/server/api.py:186]: Dunno, should be Any or None. Removing it for now

**Pull request merged**